src/View/Helper/ToolbarHelper.php

```diff
@@ -77,6 +88,9 @@ public function makeNeatArray($values, $openDepth = 0, $currentDepth = 0, $doubl
         if (empty($values)) {
             $values[] = '(empty)';
         }
+        if ($this->sort && is_array($values)) {
+            ksort($values);
+        }
```

Comment:
yup it was, but if you think it should do it only on top level i can change it or even add another checkbox for deep sorting